src/views/Verbose.vue

```diff
@@ -71,6 +71,41 @@
             v-model="filter"
             :placeholder="$t('verbose.filterPlaceholder')"
           >
+          <table>
+            <tr>
+              <td>
+                <div
+                  :class="{ 'exclude-result': true, 'selected': isExcluded('true') }"
```

Comment:
It would be good to use actual checkbox input elements here

Reply:
There aren't any other `<input type="checkbox"...` in the codebase, which is why I used the same code from the Node.vue file

Reply:
Then I am an idiot and you shouldn't follow my every step 🤪 
It's not very accessible, that's all.